doc/ChangeLog

```diff
@@ -1,3 +1,16 @@
+Changes from version 0.28.1 to 0.28.2
+-------------------------------------
+
+Release Notes:
+https://github.com/Exiv2/exiv2/issues/2914
```

Comment:
Replace w/ https://github.com/Exiv2/exiv2/milestone/13?closed=1 ?

Reply:
I've added it. I've kept the link to the issue though, because it might be useful for discussion if anything goes wrong.